src/spec.cr

```diff
@@ -118,6 +118,9 @@ if ENV["SPEC_VERBOSE"]? == "1"
   Spec.override_default_formatter(Spec::VerboseFormatter.new)
 end
 
-Signal::INT.trap { Spec.abort! }
+{% unless flag?(:win32) %}
+  # TODO(windows): re-enable this once Signal is ported
```

Comment:
I don't think there is any precedent for `TODO(windows)` notation. Is this really a good idea?

`TODO: (windows)`?  should be easier to parse.

Reply:
`TODO(windows)` is easier to grep than `/TODO:?\s+(windows)/`

Reply:
AFAIK it is general consensus that the identifier is followed by either a colon or whitespace. Adding parenthesis in between won't match `/TODO[: ]/` or similar regular expressions which are pretty commonly used for this kind of thing.

Reply:
just grep TODO, i've never seen anyone make the regex more complicated

Reply:
Take a look at tools like `rake notes` and similar. They expect a certain format, and often it includes the name followed by colon or whitespace.

I think it is a good idea to add context to comment annotations. But it shouldn't add additional complexity to the simple `# TODO: message` format. Just add the context tag as part of the message and everything is fine.

Reply:
The format I used is common, I didn't invent it. I'm sick and tired of making nitpicky corrections, especially when they're hypotheticals about fucking todo comments.

Reply:
I've never encountered this format and specifically asked for precedents. You didn't mention anything about it being used elsewhere until now.

Reply:
@straight-shoota I've seen `TODO(username): message` format before, see for instance docs for https://github.com/pgilad/leasot or https://github.com/asticode/go-astitodo. Usually what's put inside of brackets is a _username_ but that's just a convention. It's noticably less common than `TODO: message` but still.